plugin/setup.php

```diff
@@ -7,7 +7,7 @@
 use GlpiPlugin\Iservice\Utils\SpecialViewsMenu;
 use GlpiPlugin\Iservice\Utils\IserviceMenu;
 
-define('ISERVICE_VERSION', '3.0.9');
+define('ISERVICE_VERSION', '3.0.91');
```

Comment:
We use consecutive numbering
```suggestion
define('ISERVICE_VERSION', '3.0.10');
```